ipaserver/install/service.py

```diff
@@ -209,7 +209,7 @@ def _ldap_mod(self, ldif, sub_dict=None, raise_on_err=True,
 
         if dm_password:
             [pw_fd, pw_name] = tempfile.mkstemp()
-            os.write(pw_fd, dm_password)
+            os.write(pw_fd, dm_password.encode())
```

Comment:
Could you make also this code more Pythonic and replace `mkstemp()` with `TemporaryFile()`?